talend-component-maven-plugin/src/main/java/org/talend/sdk/component/maven/ValidateComponentMojo.java

```diff
@@ -149,6 +150,18 @@
     @Parameter(defaultValue = "true", property = "talend.validation.noFinalOption")
     private boolean validateNoFinalOption;
 
+    /**
+     * Should the exceptions thrown be validated
+     */
+    @Parameter(defaultValue = "true", property = "talend.validation.exceptions")
+    private boolean validateExceptions;
+
+    /**
+     * Should build fail on exception validation error
+     */
+    @Parameter(defaultValue = "true", property = "talend.validation.failOnValidateExceptions")
```

Comment:
For now, just put it at false, it will ruin builds.

Reply:
done